scope.go

```diff
@@ -1419,3 +1425,12 @@ func (scope *Scope) hasConditions() bool {
 		len(scope.Search.orConditions) > 0 ||
 		len(scope.Search.notConditions) > 0
 }
+
+func (scope *Scope) IsOracle() bool {
+	oraModules := []string{"godror", "oci8", "ora"} // must be an asc sorted slice
```

Comment:
Can this be simplified if [`GetName`](https://github.com/jimlambrt/gorm/blob/57dda4b080cb2d28c75b6b9c677ffb2fabad9e45/dialects/oracle/oracle.go#L38-L40) was changed?

```go
func (oracle) GetName() string {
	return "oracle"
}
```

Reply:
fixed